bin/gftools-check-sandbox-family.py

```diff
@@ -0,0 +1,85 @@
+"""Make a diff/gif image of a site by swapping the GF production families
+with same families hosted on the sandbox server.
+
+
+Usage:
+gftools check-sandbox-family https://www.somesite.com
+"""
+import os
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options 
```

Comment:
Is this added to the root project `requirements.txt`?

Reply:
Not yet